pykit/tests/test_command.py

```diff
@@ -1,32 +1,32 @@
 import unittest
 import os
 from pykit.transport.http import connect
-from pykit import Command
+from pykit import ClCommand
 
 
-class TestCommand(unittest.TestCase):
+class TestClCommand(unittest.TestCase):
     def setUp(self):
         self.connection = connect(
             os.environ['PK_DB2SOCK_URL'],
             db2sock_auth=(
                 os.environ['PK_DB2SOCK_USER'], os.environ['PK_DB2SOCK_PASS']))
 
     def test_execute_qsh_command(self):
-        cmd = Command("ls -1 /QOpenSys", True)
+        cmd = ClCommand("wrkactjob", True)
         toolkit = self.connection.toolkit()
         toolkit.add(cmd)
         response = toolkit.execute()
         self.assertTrue(response)
 
     def test_execute_rexx_command(self):
-        cmd = Command("RTVJOBA CCSID(?N) USRLIBL(?) SYSLIBL(?)")
+        cmd = ClCommand("RTVJOBA CCSID(?N) USRLIBL(?) SYSLIBL(?)")
         toolkit = self.connection.toolkit()
         toolkit.add(cmd)
         response = toolkit.execute()
         self.assertTrue(response)
```

Comment:
I'd like to see the test do more than just test that it executed. We should maybe ensure that the keys exist in the JSON output or something at least. I think it would be hard to test the values.

Reply:
Open an issue please.